free-3.md

```diff
@@ -4,40 +4,86 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
-
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Unusual music listeners, like myself, generally listen to a large selection
+of music that cannot be found in one place. For instance, certain songs I like
+are only found on YouTube, while others can be streamed on Spotify, listened
+to from a file, or streamed via internet radio. Although there are ways to
+download songs from nearly any source, sometimes I just want to add a song to
+a playlist without having to go through all the trouble. Unfortunately, there
+doesn't seem to good ways to create playlists that span multiple sources.
+Either you have a playlist on YouTube, Spotify, or locally, but never all
+three. This DSL will be a wrapper around a database that allows you to
+catalog a wide range of music from various sources and either play them locally
+or stream them as a personal internet radio station to a private device.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+By wrapping a DSL around annoying music codecs, music sources, and APIs,
+a single DSL will allow users to achieve many of the distinct properties
+of a playlist in a program like iTunes, without needing to have all the
+sources match. Additionally, by wrapping the DSL around streaming services,
+your collection can be easily listened to on the go via typical internet
+radio streaming services.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+This is a practical idea that my roommate has previously attempted to address
+(at least in the YouTube domain), with a tool he calls
+[Snackbar](https://www.cs.hmc.edu/~wwong/playlist). While this tool is
+exciting, I now want to make an "improved" version that also enables typical
+playlist features, streaming, and inclusion of local files. I would use this
+tool so much it isn't even funny.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Listening from multiple audio sources.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+In some ways, I am torn about the best way to address this question. On one
+hand, like ImageOrganizer, I am interested in designing this DSL so it is
+easily incorperated into custom GUIs. That being said, a command line music
+tool doesn't feel ideal. That being said, it is possible I could design it
+as a command line tool and integrate it into a GUI as a plugin to something
+like [Foobar2000](https://www.foobar2000.org/), arguably the most customizable
+media player around.
```

Comment:
I think this is the right way to go about it. I think doing it this way will lend itself to more thinking about language features and design, and less about UI and other "fluff."